src/UbiHttp.cpp

```diff
@@ -134,21 +139,25 @@ double UbiHTTP::get(const char *device_label, const char *variable_label) {
     return ERROR_VALUE;
   }
 
-  uint16_t pathLength = _pathLength(device_label, variable_label);
-  char *path = (char *)malloc(sizeof(char) * pathLength + 1);
+  int pathLength = snprintf(NULL, 0, "/api/v1.6/devices/%s/%s/lv", device_label, variable_label);
+  char *path = (char *)malloc(pathLength + sizeof(""));
   sprintf(path, "/api/v1.6/devices/%s/%s/lv", device_label, variable_label);
 
   if (_debug) {
     Serial.print(F("Requesting to URL: "));
     Serial.println(path);
   }
 
-  uint16_t requestLineLength = _requestLineLength(path);
-  char *message = (char *)malloc(sizeof(char) * requestLineLength + 1);
+  int requestLineLength = snprintf(NULL, 0,
```

Comment:
I would prefer to just replace this line where `_requestLineLenght` is defined as it is already being reused, also the implementation already has a description.

Reply:
The change is reverted.